### ISSUE TITLE
Dragons should stop growing on stage 3.   4,5 should be reached through player assistance + other ideas

### ISSUE THREAD

**Issue by taisong0612** [AlexModGuy/Ice-and-Fire-Suggestions#390]
After seeing my dragon grow to stage 5, I felt this as too easy to reach compared to its features.

Stage 5 dragons are capable of leveling entire dungeons, murdering hundreds of mobs, and generally BIG. This BIGness and EPICness is very nice, but I think there has to be specific ways for it to be accessible.
People should not be able to AFK in a box to wait for their dragon to reach stage 5.
Also, I want to do something for dragons, not just ride them like horses.

Therefore, I think stage 4 and 5 should be only available to players who defeated multiple dragons already.
Right now, most of the items are unused. I am sure this will be alleviated in the future, but I have a particular use for hearts. Currently we dont use hearts at all. So, lets use hearts to make a special meal for dragons to start the process for growing to the next stage.

Tl;dr 
Make a special meal out of dragon heart (hearts taken from the dregs of opposite aspect)

Natural Dragon growth stops at stage 3 (large enough anyway)

stage 3 Dragon starts growing to stage 4 with a special meal

Stage 4 dragon starts growing to stage 5 with a special meal

**Comment by zucaluca** [comment]
Yes
